publications-2018.bib

```diff
@@ -588,7 +588,7 @@ @Article{Potschka2018a
 @Article{castelli2018numerical,
   author    = {G.F. Castelli and W. D\"{o}rfler},
   title     = {The numerical study of a microscale model for lithium-ion batteries},
-  journal   = {Computers \& Mathematics with Applications},
+  journal   = {Computers \& Mathematics with Applications, in press},
```

Comment:
Would you mind please changing this to a
```
note = {In press},
```
There's a note about this in the README. If you think that this is a pedantic request, then the README should be updated to reflect how we'd like to consistently enter this information into all entries.

Reply:
Ah, ok. I hadn't looked through that yet.